lib/cocoapods/installer/target_installer/pod_target_installer.rb

```diff
@@ -262,6 +275,14 @@ def create_module_map
           FileUtils.cp(custom_module_map, path)
           add_file_to_support_group(path)
 
+          unless target.requires_frameworks?
+            File.open(path) do |source_file|
+              contents = source_file.read
+              contents.gsub!(/^\s*framework\s*module/, 'module')
```

Comment:
Actually, there can be multiple multiple module declarations in module map. http://clang.llvm.org/docs/Modules.html#module-map-file